transaction-status/src/parse_bpf_loader.rs

```diff
@@ -187,6 +187,7 @@ pub fn parse_bpf_upgradeable_loader(
                 }),
             })
         }
+        UpgradeableLoaderInstruction::Migrate => unimplemented!(),
```

Comment:
same here, this would cause rpc to panic. Return `ParseInstructionError::InstructionNotParsable(ParsableProgram::BpfUpgradeableLoader)` here instead

Reply:
Done with 8eed5abc8f